From 7ce9cc303e56c703576b034bb4b54a8e8249e074 Mon Sep 17 00:00:00 2001
From: Max Sakharov <maksym.sakharov@gmail.com>
Date: Sun, 8 Nov 2015 23:55:53 +0200
Subject: [PATCH] add

---
 README.md | 1 +
 1 file changed, 1 insertion(+)

diff --git a/README.md b/README.md
index a15e375..1536720 100644
--- a/README.md
+++ b/README.md
@@ -1,3 +1,4 @@
 # test
 This is test project by Max
 --------------------
+--------------------
-- 
2.6.2

